content/en/docs/zero-code/java/agent/sdk-configuration.md

```diff
@@ -0,0 +1,112 @@
+---
+title: SDK configuration
+weight: 10
+aliases: [agent-config]
+# prettier-ignore
+cSpell:ignore: akka armeria classloaders couchbase Customizer datasource dbcp Dotel dropwizard dubbo enduser finatra hikari hikaricp HSET httpasyncclient httpclient hystrix jaxrs jaxws jedis jodd kotlinx logback logmanager mojarra myfaces okhttp oshi pekko rabbitmq ratpack rediscala redisson restlet rocketmq serverlessapis spymemcached twilio vaadin vertx vibur webflux webmvc
+---
+         
+## Agent Configuration
+
+The agent can consume configuration from one or more of the following sources
+(ordered from highest to lowest priority):
+
+- system properties
+- [environment variables](#configuring-with-environment-variables)
+- the [configuration file](#configuration-file)
+- properties provided by the
```

Comment:
I know that this was in the original text, but here are some slight copyedits nonetheless:

```suggestion
- System properties
- [Environment variables](#configuring-with-environment-variables)
- [Configuration file](#configuration-file)
- Properties provided by the
```